public/js/index.js

```diff
@@ -942,6 +942,20 @@ function init_postform_features() {
       });
     });
   }
+
+  // init file pasting
+  if (post_form != null) {
+    let postform_message = document.getElementById('form-post-message');
+    postform_message.addEventListener('paste', (event) => {
+      if (event.clipboardData.files.length > 0) {
+        event.preventDefault();
+        let fileInput = document.getElementById('form-file');
+        if (fileInput !== null) {
```

Comment:
This should probably be != instead? So that it catches undefined and null?

Reply:
getElementById returns null if no element found by ID and strict types is good practice.

Reply:
I guess it's okay, but I've always favored != to check for null and undefined. Even though they are not the same thing, I've treated them similarly, but okay fine, mostly a subjective opinion.